### ISSUE TITLE
对学习率衰减支持得不好

### ISSUE THREAD

**Issue by hhr114** [lyhue1991/torchkeras#39]
首先感谢开源这么有意义的项目！

发现torchkeras对对学习率衰减支持得不好，首先是lr_scheduler.step()没传入参数metrics，其次是使用lr_scheduler后，学习率明明已经下降，但processing bar显示的lr仍然不变…… 改起来应该不难，希望作者更新fix

**Comment by lyhue1991** [comment]
lr_scheduler.step()没传入参数metrics是指的？
pbar显示不变是因为lr太小了，被舍入掉了，你放个0.001的学习率试试？

**Comment by hhr114** [comment]
在StepRunner的__call__下面有代码段
```python
if self.lr_scheduler is not None:
    self.lr_scheduler.step()
```
运行会报错，然后我改成了 `self.lr_scheduler.step(loss)`，现在是每个Step符合条件就更新一次学习率，但我觉得这还是不太合理，应该是每个epoch检验一次？另外这里传入的参数应该可以是metric_dict中的任一个？还需要一些完善和补充。

**Comment by lyhue1991** [comment]
这边有测试过正常的scheduler是可以跑的，你定义scheduler的代码是什么?

**Comment by hhr114** [comment]
```
model = torchkeras.KerasModel(net, loss_fn=nn.MSELoss(),
                                optimizer=torch.optim.Adam(net.parameters(), lr=args.lr),
                                lr_scheduler=ReduceLROnPlateau(torch.optim.Adam(net.parameters(), lr=args.lr), mode='min', factor=0.1, patience=1, verbose=True, min_lr=1e-5), 
                                metrics_dict={'mse':trueMSE()})
```

**Comment by lyhue1991** [comment]
ReduceLROnPlateau这个scheduler用的不多了，其他的大部分scheduler都是不需要传入loss或metric的哦，并且是每个step更新。你可以用Cosine这些别的scheduler替代一下，不然就需要自定义 StepRunner和 EpochRunner了。

**Comment by hhr114** [comment]
还有个问题，我在使用的时候，通过打印的日志发现，只有第一个epoch每个step更新使用了scheduler，后面的epoch好像都没有用到我的lr了（即使还没到我定义的scheduler最小值）

**Comment by lyhue1991** [comment]
日志只会输出每个epoch最后的lr。建议看看StepRunner和EpochRunner的源代码，不是很长，拿过来调试一下看看是不是有问题。
